oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -30,82 +30,64 @@ OEP-14: Archiving edX GitHub Repositories
 Abstract
 ========
 
-The `edx organization`_ contains a large number of repositories, most of
-which are active and maintained, but some of which are now obsolete. To
-clarify the status of repositories, a process for archiving a
-repository is defined below.
+The `edx organization`_ contains a large number of repositories, most of which are active and maintained, but some of which are now obsolete. To clarify the status of repositories, a process for archiving a repository is defined below.
 
 .. _edx organization: https://github.com/edx
 
 Motivation
 ==========
 
-Recently openedx.yaml files were added to edX repositories per `OEP-2`_. In the
-course of deciding owners for those repositories, there was an `ORA PR
-Discussion`_ about how best to handle deprecated or obsolete repositories. In
-particular, do obsolete repositories need owners, and how can repositories be
-clearly marked as present for archive purposes only?
+Recently openedx.yaml files were added to edX repositories per `OEP-2`_. In the course of deciding owners for those repositories, there was an `ORA PR Discussion`_ about how best to handle deprecated or obsolete repositories. In particular, do obsolete repositories need owners, and how can repositories be clearly marked as present for archive purposes only?
 
-This discussion resurfaced related to edX's usage of Gemnasium to report
-the usage of third-party libraries that have known security issues. All
-repositories under the edX organization were being monitored, but this added
-noise when trying to understand the number of third-party library updates
-required for actively maintained repositories.
+This discussion resurfaced related to edX's usage of Gemnasium to report the usage of third-party libraries that have known security issues. All repositories under the edX organization were being monitored, but this added noise when trying to understand the number of third-party library updates required for actively maintained repositories.
 
 .. _OEP-2: https://open-edx-proposals.readthedocs.io/en/latest/oep-0002.html
 
 
 Specification
 =============
 
-When a repository under the `edx organization`_ will no longer be maintained
-because it is no longer in use, the following steps should be followed.
+When a repository under the `edx organization`_ will no longer be maintained because it is no longer in use, the following steps should be followed.
```

Comment:
What does "no longer in use" mean in this sense? How is it determined if something is no longer in use or "will no longer be maintained"? Is this specifically in regards to features that are not being used on edx.org or things that have gone through the official deprecation process? Is this in a separate OEP that I should reference?